src/iAd/ADCompat.cs

```diff
@@ -289,22 +289,50 @@ public virtual void WillLoad(ADBannerView bannerView)
 	// some of this API is still provided
 	public partial class ADClient : NSObject {
 
-		[Obsoleted (PlatformName.iOS, 15, 0, PlatformArchitecture.None, Constants.iAdRemoved)]
```

Comment:
why to you change the existing API ?
has something changed in the headers ?

if not then the legacy attributes should remain identical - this is what will be used to review the PRs

Reply:
Since these used to just be obsolete, I should not have the deprecated nor the Unsupported right? I should just have the obsolete?

Reply:
The `[UnsuppotedOSPlatform]` with the version number is fine, it helps the new analyzer.

Reply:
However the `[Deprecated]` should not be added (unless the header changes)

and `[Obsoloted]` (with a `d` suffix) should not be used in the `NET` section.